### DISCUSSION TITLE
Can 2 AdGuard Home instances share the same working directory?

### DISCUSSION THREAD

**Discussion by nmpu** [AdguardTeam/AdGuardHome#4875] [Q&A]
I currently run 2 instances of dnsmasq each with different upstream DNS. The family LAN and NSFW networks are isolated in hardware by VLAN. In order to access the NSFW network, you need to have physical access to an associated network jack.

It looks like AdGuard stores most settings in the config file. Obviously, I'd need different config files and DNS/http listen ports.

Do I need 2 different working directories or can 1 be shared?

**Comment by ainar-g** [comment]
It is strongly recommended to use separate working directories for separate installations.  Besides the configuration file, there are other files and directories AdGuard Home is using.

**Comment by candybars1** [comment]
if this is still relevant why not share the same yaml fileand use ".0.0.0.0" both for listening on port 53 and for UI no need to sync all else can remain separate
